src/components/HyloTipTapEditor/HyloTipTapEditorMobile.js

```diff
@@ -0,0 +1,110 @@
+import React, { useState, useCallback, useEffect, useRef } from 'react'
+import { pickBy } from 'lodash/fp'
+import { WebViewMessageTypes } from 'hylo-shared'
+import { sendMessageToWebView } from 'util/webView'
+import HyloTipTapEditor from 'components/HyloTipTapEditor'
+import getQuerystringParam from 'store/selectors/getQuerystringParam'
+
+// Note: This Mobile editor can be tested in a browser at `hyloApp/editor`.
+//       To control the editor manually post messages via the Web Console, e.g.:
+//
+//       `postMessage(JSON.stringify({ type: 'SET_PROPS', data: { readOnly: true } }))`
+//
+export default function HyloTipTapEditorMobile (props) {
```

Comment:
This was the hard part. Getting the editor to work in a WebView requires re-plumbing the needed parts of Native <> WebViewEditor interaction through a POJS interface that I've largely had to invent (hence the use of shard constants). It's working. When we make changes to `HyloTipTapEditor` we must also always be aware of it's mobile use.